merlion/models/ensemble/MoE_forecast.py

```diff
@@ -285,9 +287,6 @@ def moe_model(self, moe_model):
                 self.optimiser = torch.optim.Adam(self.moe_model.parameters(), lr=self.lr, weight_decay=0.00000)
             if self.lr_sch is None:
                 self.lr_sch = LRScheduler(lr_i=0.0000, lr_f=self.lr, nsteps=self.warmup_steps, optimizer=self.optimiser)
-        else:
-            self.optimiser = None
-            self.lr_sch = None
```

Comment:
Can you add back these lines? I think this is an important check to ensure that the optimizer and LR scheduler remain consistent in case the MoE model is set to `None`.

Reply:
Sure, will do. Thanks for the explanation, it just felt redundant within the local context.